src/DocumentFormat.OpenXml/Spreadsheet/CellValue.cs

```diff
@@ -17,7 +17,7 @@ public partial class CellValue
         /// </summary>
         /// <param name="dateTime">DateTime for cell</param>
         public CellValue(DateTime dateTime)
-            : this(dateTime.ToString("o", CultureInfo.InvariantCulture))
+            : this(dateTime.ToString("yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fff", CultureInfo.InvariantCulture))
```

Comment:
I'd prefer if this was a const, ie

```cs
private const string DateTimeFormatString = "....";
private const string DateTimeOffsetFormatString = DateTimeFormatString + "zzz";
```

It would make it much easier to see that they're pretty much the same.